monitoring/bin/change_grafana_admin_password.sh

```diff
@@ -116,18 +116,24 @@ log_info "Updating the admin password using Grafana CLI"
 kubectl exec -n $namespace $grafanaPod -c grafana -- bin/grafana-cli admin reset-admin-password $password
 
 # Exit out of the script if the Grafana CLI call fails
-if (( $? == 1 )); then
+if (( $? != 0 )); then
   log_error "An error occurred when updating the password"
   exit 1
 fi
 
 # Patch new password in Kubernetes
 encryptedPassword="$(echo -n "$password" | base64)"
 log_info "Updating Grafana secret with the new password"
-kubectl -n $namespace patch secret $v4mSecret --type='json' -p="[{'op' : 'replace' ,'path' : '/data/admin-password' ,'value' : '$encryptedPassword'}]"
+kubectl -n $namespace patch secret $grafanaInstance --type='json' -p="[{'op' : 'replace' ,'path' : '/data/admin-password' ,'value' : '$encryptedPassword'}]"
 
 # Restart Grafana pods and wait for them to restart
-log_info "Restarting the Grafana pods"
-kubectl delete pods -n $namespace -l "app.kubernetes.io/instance=v4m-prometheus-operator" -l "app.kubernetes.io/name=grafana"
-kubectl -n $namespace wait pods --selector "app.kubernetes.io/instance=v4m-prometheus-operator","app.kubernetes.io/name=grafana" --for condition=Ready --timeout=2m
-log_info "Grafana admin password has been updated"
+log_info "Grafana admin password has been updated.  Restarting Grafana pods to apply the change"
+if [ "$cluster" == "true" ]; then
+    kubectl delete pods -n $MON_NS -l "app.kubernetes.io/instance=v4m-prometheus-operator" -l "app.kubernetes.io/name=grafana"
+    kubectl -n $MON_NS wait pods --selector "app.kubernetes.io/instance=v4m-prometheus-operator","app.kubernetes.io/name=grafana" --for condition=Ready --timeout=2m
```

Comment:
Hmmm, referring to $MON_NS here again...and then to $tenantNS in line 137...can't you just use $namespace here (just like the change you just made above)?

Reply:
That was a goof-up, too.  It'll be addressed in the next push.